### PR TITLE
Flappy avatars

### PR DESCRIPTION
**Pull request by Atlante45**
Entity script that makes a small mini game.

I tried to design that script so that it was highly reusable for other mini games.
Unfortunately by adding more and more features, it kinda cramped the code.
So I think I'll go back and clean that up later. Probably for the next mini game I'll make.

### CODE REVIEW

**Comment by Atlante45** [comment]
You can import this file, this will spawn an entity you can equip.
Then use the trigger to play.
[flappyAvatars.json.txt](https://github.com/highfidelity/hifi/files/157965/flappyAvatars.json.txt)

**Comment by hifi-gustavo** [comment]
Build console output for commit: 10d55666f2af067a127729d566482ee7311993c2 available at: https://gist.github.com/874f30c943225c24218f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0364703f86817a5e8249893f55a4d01a500bf44e available at: https://gist.github.com/874f30c943225c24218f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.dmg

**Comment by thoys** [comment]
f4e0c480d477f75fc7ba83330090d0c2

Tested with hydra's. After 20 attempts I set a highscore of 9. Sounds work aswell. Avatar becomes the avatar of the user. 

Great fun, QA passed! :white_check_mark:

File: examples/toybox/flappyAvatars.js [hunk 1]
```diff
@@ -0,0 +1,730 @@
...
+//
+
+(function() {
+	// Constants
+    var TRIGGER_CONTROLS = [
+        Controller.Standard.LT,
+        Controller.Standard.RT,
+    ];
+
+	var G = 4.0;
```
Comment (ZappoMan) [inline examples/toybox/flappyAvatars.js]: formatting is messed up in this file. Looks like you have tabs instead of 4 spaces.

**Comment by Atlante45** [comment]
Dammit sublime, "Y U Not use spaces!".
Fixing.

File: examples/toybox/flappyAvatars.js (hunk 1 above)
Comment (Atlante45) [inline examples/toybox/flappyAvatars.js]: Dammit sublime, "Y U NO use spaces!".
Fixing.

**Comment by hifi-gustavo** [comment]
Build console output for commit: f62a3ed8f6a94733c1f843473bbed0c5c88be111 available at: https://gist.github.com/874f30c943225c24218f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: ab5a29b076a37479295061bc764fcadaf8bea470 available at: https://gist.github.com/874f30c943225c24218f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 3a5165ab6642c8f46dac6ed2886e69f9924b5fb8 available at: https://gist.github.com/874f30c943225c24218f<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7244.dmg

**Pull request merged**